Projects/UOContent/Items/Skill Items/Magical/Runebook.cs

```diff
@@ -263,7 +263,7 @@ public override void OnDoubleClick(Mobile from)
             }
 
             from.CloseGump<RunebookGump>();
-            from.SendGump(new RunebookGump(from, this));
+            from.SendGump(new RunebookGump(this));
 
             Openers.Add(from);
```

Comment:
Instead of this, we should make a convenience function and call it from OnResponse from situations where the gump should be opened again.

Reply:
so I'm going to make this a static `Resend` function on the gump so the `InternalPrompt` can call it effectively, lmk if this is dumb